lab-bulma-react/src/App.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+
+// Navbar
+
+function Navbar() {
```

Comment:
Better if you create a file for each of the components instead of putting all of them in one unique file